README.md

```diff
@@ -81,13 +81,23 @@ ETNA supports configuration files. It means that library will check that all the
 
 To set up a configuration for your project you should create a `.etna` file at the project's root. To see the available options look at [`Settings`](https://github.com/tinkoff-ai/etna/blob/master/etna/settings.py#L68). There is an [example](https://github.com/tinkoff-ai/etna/tree/master/examples/configs/.etna) of configuration file. 
 
-## Get started 
+## Get started
+
 Here's some example code for a quick start.
 ```python
 import pandas as pd
+
 from etna.datasets.tsdataset import TSDataset
-from etna.models import ProphetModel
+from etna.metrics import SMAPE
+from etna.models import CatBoostModelMultiSegment
 from etna.pipeline import Pipeline
+from etna.transforms import DateFlagsTransform
+from etna.transforms import DensityOutliersTransform
+from etna.transforms import FourierTransform
+from etna.transforms import LagTransform
+from etna.transforms import LinearTrendTransform
+from etna.transforms import SegmentEncoderTransform
+from etna.transforms import TimeSeriesImputerTransform
```

Comment:
Can we reduce the number of transforms?

Reply:
I thought that we want to demonstrate many of them. How much do we want to see?

Reply:
This combination of transform gave me a good score. Some of transforms have been removed because of this reason.